ext/include/opentelemetry/ext/http/client/http_client.h

```diff
@@ -227,8 +227,11 @@ class HttpClient
 {
 public:
   virtual std::shared_ptr<Session> CreateSession(nostd::string_view host,
-                                                 uint16_t port = 80) noexcept = 0;
-  virtual bool CancelAllSessions() noexcept                                   = 0;
+                                                 uint16_t port) noexcept = 0;
+
+  virtual std::shared_ptr<Session> CreateSession(nostd::string_view url) noexcept = 0;
```

Comment:
Okay, I see what you are doing. But I am not sure - maybe we should just deprecate the first method?

Reply:
Yes exactly, I just overloaded the function because that is what the @lalitb (the original issue creator) said might have been good. If its alright I can just simply remove the old parameter and just leave the new one since its also consistent with the `GET` and `POST` commands. But that is definitely a decision for the maintainers so when they give their input we can see whats the best course of action. @lalitb  @reyang

Reply:
Yes, it should be fine removing the original existing method as @maxgolov suggested.